app/models/miq_replication_worker/runner.rb

```diff
@@ -36,7 +36,7 @@ def valid_destination_settings?
     dest_conf = worker_settings.fetch_path(:replication, :destination)
     dest_conf[:adapter] ||= "postgresql"
     region = MiqRegion.my_region
-    host_user_pass = dest_conf.values_at(:host, :port, :username, :password, :database, :adapter)
+    host_user_pass = dest_conf.values_at(:host, :username, :password, :database, :adapter)
     host_user_pass.none?(&:blank?) && MiqRegionRemote.region_valid?(region.guid, region.region, *host_user_pass)
```

Comment:
Or maybe said another way...why isn't the UI requiring on a port number / why is the UI allowing the port number to be blanked out...we give a default value, so it's not like it needs to be guessed by the user.